libkbfs/connection_status.go

```diff
@@ -10,10 +10,21 @@ import (
 
 // Service names used in ConnectionStatus.
 const (
-	KeybaseServiceName = "keybase-service"
-	MDServiceName      = "md-server"
+	KeybaseServiceName     = "keybase-service"
+	MDServiceName          = "md-server"
+	LoginStatusUpdateName  = "login"
+	LogoutStatusUpdateName = "logout"
 )
 
+// alwaysPushConnectionStatus is pushed every time to Status receivers,
```

Comment:
Ah, okay, now I understand how this is used.

It seems to me we can avoid the need for adding these placeholder names and errors, though. I suggest:

1) Adding a `PushStatusChange()` method to `kbfsCurrentStatus`, which just does:

```
	kcs.lock.Lock()
	defer kcs.lock.Unlock()
	close(kcs.invalidateChan)
	kcs.invalidateChan = make(chan StatusUpdate)
```

2) Add a similar method to `KBFSOps` (you'll have to run `gen_mocks.sh` from `libkbfs` after doing so)

3) Use that instead of `PushConnectionStatusChange()` with `alwaysPushConnectionStatus`.